pkg/util/iptables/iptables_test.go

```diff
@@ -1198,3 +1198,152 @@ func TestRestoreAllWaitBackportedIptablesRestore(t *testing.T) {
 		t.Errorf("expected failure")
 	}
 }
+
+// TestExtractLines tests that
+func TestExtractLines(t *testing.T) {
+	type args struct {
+		count int
+		line  int
+		lines []byte
+	}
+	lines := "Line0: 0\nLine1: 1\nLine2: 2\nLine3: 3\nLine4: 4\nLine5: 5\nLine6: 6\nLine7: 7\nLine8: 8\nLine9: 9\nLine10: 10"
+	tests := []struct {
+		name string
+		args args
+		want []LineData
+	}{
+		{
+			"test0",
+			args{
+				count: 0,
+				line:  0,
+				lines: []byte(lines),
+			},
+			[]LineData{
+				{
```

Comment:
This might be the right place to NOT use keys.  `LineData{0, "Line0: 0"}` is one line instead of 4

Reply:
In fact:

```
mkLines := func(lines ...LineData) []LineData {
    return lines
}

// then use it as:

want: mkLines(LineData{0, "line0"}, LineData{1, "line1"}),
```

easier on the eyets